src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -451,8 +449,8 @@ void CompositionalMultiphaseBase::updateFluidModel( Group & dataGroup, localInde
 {
   GEOSX_MARK_FUNCTION;
 
-  arrayView1d< real64 const > const pres = dataGroup.getReference< array1d< real64 > >( viewKeyStruct::pressureString() );
-  arrayView1d< real64 const > const dPres = dataGroup.getReference< array1d< real64 > >( viewKeyStruct::deltaPressureString() );
+  arrayView1d< real64 const > const pres = dataGroup.getReference< array1d< real64 > >( extrinsicMeshData::pressure::key() );
+  arrayView1d< real64 const > const dPres = dataGroup.getReference< array1d< real64 > >( extrinsicMeshData::deltaPressure::key() );
```

Comment:
Is there a reason why you do not use `getExtrinsicData` here (and in other places of the code) while you use in other places?

Reply:
Yes there is a good reason, sorry, I am not done with the compositional multiphase solvers :) I am working on them now

Reply:
HA HA, sorry I should have waited!

Reply:
It should be good now. I went through the single-phase and compositional solvers, and replaced as many `getReference` as possible (I may have forgotten a few). I will stop here, and do the wells in a separate PR.